documentation/modules/proc-deleting-users-cli.adoc

```diff
@@ -1,19 +1,12 @@
 // Module included in the following assemblies:
 //
-// <List assemblies here, each on a new line>
+// assembly-managing-users-oc.adoc
+// assembly-managing-users-kube.adoc
 
-// Base the file name and the ID on the module title. For example:
-// * file name: doing-procedure-a.adoc
-// * ID: [id='doing-procedure-a']
-// * Title: = Doing procedure A
-
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id='proc-deleting-users-cli-{context}']
-// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
 = Deleting users using the command line
 
-// Start the title of a procedure module with a verb, such as Creating or Create. See also _Wording of headings_ in _The IBM Style Guide_.
-Users can in {ProductName} can be deleted using standard command line tools.
+Users can be deleted using standard command-line tools.
```

Comment:
Why is command line hyphened here but not the title?

Reply:
Because here it's an adjective, but in the title it's a noun (adjectives get hyphens)